server.ts

```diff
@@ -68,15 +78,28 @@ app.post("/", async (req, res) => {
     let sendTxId: string;
     try {
         let inputs: slpjs.SlpAddressUtxoResult[] = [];
-        inputs = inputs.concat(changeAddr.balance.slpTokenUtxos[process.env.TOKENID!]).concat(changeAddr.balance.nonSlpUtxos);
+        if (process.env.NFT === 'yes') {
```

Comment:
Same here, handle nft logic in slpfaucet.  Create a new method in slpfaucet to pick which send method to call.  This will keep the server.ts as clean as possible.